snowplow/iglu-client-embedded/schemas/com.metabase/search/jsonschema/1-0-1

```diff
@@ -0,0 +1,89 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Search events",
+  "self": {
+    "vendor": "com.metabase",
+    "name": "search",
+    "format": "jsonschema",
+    "version": "1-0-1"
+  },
+  "type": "object",
+  "properties": {
+    "event": {
+      "description": "Event name",
+      "type": "string",
+      "enum": [
+        "new_search_query",
+        "search_results_filtered"
+      ],
+      "maxLength": 1024
+    },
+    "runtime_milliseconds": {
+      "description": "Number of milliseconds it took to successfully run the search query",
+      "type": [
+        "integer",
+        "null"
+      ],
+      "minimum": 0,
+      "maximum": 2147483647
+    },
+    "content_type": {
+      "description": "Filter for entity types (question, database, model, etc.)",
+      "type": [
+        "array",
+        "null"
+      ],
+      "items": {
+        "description": "An entity type (card, question, dashboard, etc.)",
+        "type": "string",
```

Comment:
Wdyt about adding a enum here? Otherwise we can easily get incorrect results in Redshift

Reply:
yes! good call

Reply:
Added

Reply:
I [updated the spec](https://www.notion.so/metabase/Search-results-filtered-f64a868f3901451ead4187f26bf9a5a3) to make sure they're in sync